src/Microsoft.OpenApi/Validations/Rules/OpenApiResponsesRules.cs

```diff
@@ -14,16 +14,44 @@ namespace Microsoft.OpenApi.Validations.Rules
     public static class OpenApiResponsesRules
     {
         /// <summary>
-        /// An OpenAPI operation must contain at least one successful response
+        /// An OpenAPI operation must contain at least one response
         /// </summary>
-        public static ValidationRule<OpenApiResponses> ResponsesMustContainSuccessResponse =>
+        public static ValidationRule<OpenApiResponses> ResponsesMustContainAtLeastOneResponse =>
             new ValidationRule<OpenApiResponses>(
-                (context, item) =>
+                (context, responses) =>
                 {
-                    if (!item.Keys.Any(k => k.StartsWith("2"))) {
-                        context.AddError(new ValidationError(ErrorReason.Required,context.PathString,"Responses must contain success response"));
+                    if (!responses.Keys.Any())
+                    {
+                        context.AddError(
+                            new ValidationError(
+                                ErrorReason.Required,
+                                context.PathString,
+                                "Responses must contain at least one response"));
                     }
                 });
 
+        /// <summary>
+        /// The response key must either be "default" or an HTTP status code (1xx, 2xx, 3xx, 4xx, 5xx)
+        /// </summary>
+        public static ValidationRule<OpenApiResponses> ResponsesMustBeIdentifiedByDefaultOrStatusCode =>
+            new ValidationRule<OpenApiResponses>(
+                (context, responses) =>
+                {
+                    foreach (var key in responses.Keys)
+                    {
+                        context.Enter(key);
+
+                        if (key != "default" && !Regex.IsMatch(key, "^[1-5|X][0-9|X][0-9|X]$"))
```

Comment:
The spec is kinda ambiguous here on whether XXX is allowed. I can see that the intention is probably "not allowed" - since "default" is created for that purpose. However, the exact wording of the spec itself does not make it crystal clear which digits the wildcard X can be used in.

To define a range of response codes, this field MAY contain the uppercase wildcard character X. For example, 2XX represents all response codes between [200-299]. The following range definitions are allowed: 1XX, 2XX, 3XX, 4XX, and 5XX. If a response range is defined using an explicit code, the explicit code definition takes precedence over the range definition for that code.

Reply:
I submitted a PR to the spec to clarify the wording.  Does [this](https://github.com/OAI/OpenAPI-Specification/pull/1489/files?short_path=18985dd#diff-18985dd9fd564e6d6dbe0c6cb887484d) help?

Reply:
Yup. That helps. I'll fix my code